cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/resource/StatefulSetDiff.java

```diff
@@ -27,10 +27,12 @@
     static {
         IGNORABLE_PATHS = asList(
             "/spec/revisionHistoryLimit",
-            "/spec/template/spec/initContainers/0/imagePullPolicy",
-            "/spec/template/spec/initContainers/0/resources",
-            "/spec/template/spec/initContainers/0/terminationMessagePath",
-            "/spec/template/spec/initContainers/0/terminationMessagePolicy",
+            "/spec/template/spec/initContainers/[0-9]+/imagePullPolicy",
+            "/spec/template/spec/initContainers/[0-9]+/resources",
+            "/spec/template/spec/initContainers/[0-9]+/terminationMessagePath",
+            "/spec/template/spec/initContainers/[0-9]+/terminationMessagePolicy",
+            "/spec/template/spec/initContainers/[0-9]+/env/[0-9]+/valueFrom/fieldRef/apiVersion",
```

Comment:
Why are you ignoring the changes to environment variables? This will basically mean that changes to the topology key will be ignored, or? I think this is wrong. Can you share more details about what is in these fields, why is it changing and why should we ignore them?

Reply:
I should re-check but I notice that the the "current" has apiVersion v1 while the "desired" has null. We don't set this apiVersion on SS creation but it seems that K8S/OS adds this value. It's not a real difference for this reason I ignored it.